quarkchain/evm/specials.py

```diff
@@ -226,6 +226,30 @@ def proc_ecpairing(ext, msg):
     return 1, msg.gas - gascost, [0] * 31 + [1 if result else 0]
 
 
+# 3 inputs: (address, token ID and value)
+def proc_transfer_mnt(ext, msg):
+    print("transfer_mnt proc", msg.gas)
+    gascost = 3  # to be discussed
+    if msg.gas < gascost:
+        return 0, 0, []
+    to = utils.int_to_addr(msg.data.extract32(0))
```

Comment:
missing data size check?

Reply:
Check added to make sure `data.size` is exactly 3 * 32 byte